_data/glossary.yml

```diff
@@ -0,0 +1,31 @@
+canonical-tags:
+- Fundamentals
+- K8s Objects
+- Metadata
+- Configuration
+- Security
+- Networking
+- Storage
+- Cluster Operation
+- Workloads
+- Architecture
```

Comment:
What about Components? https://kubernetes.io/docs/concepts/overview/components/

I tried to think of other synonyms, but I think that is the most fitting.